subsystem/host_test.go

```diff
@@ -692,17 +692,24 @@ var _ = Describe("Host tests", func() {
 
 		// register again to cluster 2 and expect it to be in discovery status
 		Expect(db.Model(h).Update("status", "known").Error).NotTo(HaveOccurred())
```

Comment:
lets remove all this part.
It can cause a race with the monitor and it doesn't really check anything.